### ISSUE TITLE
Checking if related User has a valid token

### ISSUE THREAD

**Issue by paulo-hortelan** [shetabit/tokenable#6]
Is there a method to, without the token, check if the related User has any valid token? In case not and you believe that this would be a nice new feature, I could do a pull request

**Comment by khanzadimahdi** [comment]
Hi Paulo, thanks for creating this issue.
what would be the use case of this feature?

**Comment by paulo-hortelan** [comment]
I need to know if the user has any valid token before allowing him to create a new one. Basically, is the token/pin `Forgot password`. 

Right now I'm looping through the users related tokens and checking if if he has one valid but having a method that can do that would be great.

**Comment by khanzadimahdi** [comment]
I agree that it's a good idea to add functionality to get user's valid tokens by type.

could you please create a PR for that?
